docs/onboarding/azure/ad-entra-setup/powershell/aad-app-registration.ps1

```diff
@@ -87,6 +87,27 @@ az ad app permission admin-consent --id $appId
 Write-Host "Generating app credential..." -ForegroundColor Blue
 $password = az ad app credential reset --id $appId --append --end-date $endDate --query password -o tsv
 
+# Define the JSON in a PowerShell object
+Write-Host "Preparing optional claims JSON..." -ForegroundColor Blue
+$optionalClaims = @{
+  idToken = @(
+    @{
+      name = "email"
+      essential = $false
```

Comment:
Does this code fragment configure the App's sign-on to include `email` claim in id tokens?  If so, should we set `essential` to `$true`?